crates/rpc/bridge/src/lib.rs

```diff
@@ -15,3 +21,39 @@ pub trait StrataBridgeControlApi {
     #[method(name = "uptime")]
     async fn get_uptime(&self) -> RpcResult<u64>;
 }
+
+#[cfg_attr(not(feature = "client"), rpc(server, namespace = "stratabridge"))]
+#[cfg_attr(feature = "client", rpc(server, client, namespace = "stratabridge"))]
+pub trait StrataBridgeMonitoringApi {
+    /// Get all bridge operator IDs
+    #[method(name = "bridgeOperators")]
+    async fn get_bridge_operators(&self) -> RpcResult<Vec<OperatorIdx>>;
```

Comment:
This returns a 
https://github.com/alpenlabs/strata-bridge/blob/24a0e87391aaf0ff240b718cb4a7d23ea0b607e9/crates/primitives/src/types.rs#L10

How useful is that?
Isn't it better to have?
```suggestion
    async fn get_bridge_operators(&self) -> RpcResult<PublicKeyTable>;
```
due to the fact that this gives much more information? https://github.com/alpenlabs/strata-bridge/blob/24a0e87391aaf0ff240b718cb4a7d23ea0b607e9/crates/primitives/src/types.rs#L13-L19

Reply:
Yes. I asked @Rajil1213 if public key is needed for any of the API calls. Apparently not. I am fine either way.